config/samples/v2/kustomization.yaml

```diff
@@ -0,0 +1,2 @@
+resources:
+- dspa_simple.yaml
```

Comment:
given the other pipelines are contained in their own directories, should we do the same for `dspa_simple.yaml` and `dspa_all_fields.yaml` for consistency?

Reply:
I didn't do that to avoid changing too many files. But I agree it would be better. I'll do it.

Reply:
Done.